lerna.json

```diff
@@ -1,16 +1,21 @@
 {
-  "packages": [
-    "packages/*"
-  ],
-  "version": "12.1.1",
-  "command": {
-    "bootstrap": {
-      "hoist": true
+    "packages": [
+        "packages/*"
+    ],
+    "version": "12.1.1",
+    "command": {
+        "bootstrap": {
+            "hoist": true
+        },
+        "version": {
+            "conventionalCommits": true,
+            "changelogPreset": "conventionalcommits",
```

Comment:
I think we are missing the dependency.

Reply:
Do you mean `@commitlint/config-conventional`?

Reply:
Nop, I mean the one from conventional changelog. Grab it from pico.

Reply:
Done ✅